include/private/gcconfig.h

```diff
@@ -1310,6 +1316,23 @@ EXTERN_C_BEGIN
 #     if defined(GC_THREADS) && !defined(CPPCHECK)
 #       error No threads support yet
 #     endif
+#     ifdef EMSCRIPTEN
+#       define OS_TYPE "EMSCRIPTEN"
+#       define DATASTART (ptr_t)ALIGNMENT
+#       define DATAEND (ptr_t)ALIGNMENT
+#     endif
+#     ifdef WASI
+        extern unsigned char __global_base;
```

Comment:
I think "unsigned" is redundant here. Also, please keep them more close to DATASTART/END